window-decoration/kdecoration/hellobutton.h

```diff
@@ -139,6 +139,7 @@ namespace Hello
         const QRgb colorClose = 0xFFF25056;
         const QRgb colorMaximize = 0xFF13D13D;
         const QRgb colorMinimize = 0xFFFCBE07;
+        const QRgb colorShade = 0xFFCCB0D5;
```

Comment:
That's not really necessary as I'm not using any of these other than `colorSymbol`. Those are remnants of the Breezeway decoration I forked for this. Need to clean that up.